components/use-cases/use-cases.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Use Cases
+permalink: /components/use-cases/
+---
+
+* TOC
+{:toc}
+
+Data templates are pre-defined sets of [component configurations](/components/). The individual configurations are connected 
+by an [orchestration](/orchestrator/) and [metadata](/storage/tables/#metadata). Together they form a functional reusable block
+solving a specific problem. 
+
+Keboola Connection is a very open [platform](/overview), which may be overwhelming in many cases. 
+The ambition of data templates is to ease the configuration of repeated (or repeatable) solutions. A
+lthough they are pre-defined sets of configurations, the created configurations have no special properties and 
+can be modified like any other configuration you have [created manually](/components/#creating-component-configuration).
+
+## Using Data Template
+To use a data template, go to the **Use Cases** section. 
+
+![Screenshot - Data Templates Intro](/components/use-cases/1.png)
```

Comment:
jo určitě potřebujem přidat 
{: .image-popup}

Reply:
hotovo